.gitignore

```diff
@@ -33,6 +33,7 @@ logs/
 /templates/
 /artifacts/
 /data/
+/preview/
```

Comment:
fixed. initially had preview dir at same level as data. now moved it inside data dir.